agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -241,4 +270,32 @@ private static final void setLasModifiedTime(NotificationInfo notification, Temp
                                                                      TimeConvertUtils.YEAR));
   }
 
+  private static final String getEventCreatorOrModifierUserName(long idUser) {
```

Comment:
Same here, the method name has to refer to what it does and not where it will be used